recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,18 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
-    movie_ratings = None
-    '''
-    complete the code to compute a variable movie_ratings
-    1. perform a merge between data_movies and data_ratings on 'movieId'.
```

Comment:
The 'response' variable is undefined. Initialize it as an empty list before line 11.